### PR TITLE
Updating the spec for clarity and parity with the implementation of the V7 transaction details

### PR DESCRIPTION
**Pull request by vivek-arte**
A quick summary of the changes here:
- The `assetDescSize` field in the issuance action description is encoded as a `compactSize` type, as is standard across the specification (and also allows for the use of the existing methods in the implementation).
- The description of the notes used inside the issuance bundle were explicitly serialized to remove ambiguity.
- The type of the `valueBurn` field was set to the standard `uint64` type.
- The order of the Orchard bundle fields was adjusted to have the binding signature after the burn fields.

### CODE REVIEW

File: zip-0230.rst [hunk 1]
```diff
@@ -317,26 +315,48 @@ An issuance action, ``IssueAction``, is the instance of issuing a specific Custo
...
++-----------------------------+--------------------------+--------------------------------------+---------------------------------------------------------------+
+|``8``                        |``value``                 |``uint64``                            |The amount of the Asset being issued in this note.             |
++-----------------------------+--------------------------+--------------------------------------+---------------------------------------------------------------+
+|``32``                       |``assetBase``             |``byte[32]``                          |The encoding of the Asset Base                                 |
+|                             |                          |                                      |:math:`\mathsf{AssetBase^{Orchard}}\!`.                        |
++-----------------------------+--------------------------+--------------------------------------+---------------------------------------------------------------+
+|``32``                       |``rho``                   |``byte[32]``                          |This is defined and encoded in the same manner as for          |
```
Comment (PaulLaux) [inline zip-0230.rst:353]: need to be specific about where exactly it is defined or define it here. (x2)

File: zip-0230.rst [hunk 2]
```diff
@@ -317,26 +315,48 @@ An issuance action, ``IssueAction``, is the instance of issuing a specific Custo
...
+
++-----------------------------+--------------------------+--------------------------------------+---------------------------------------------------------------+
+| Bytes                       | Name                     | Data Type                            | Description                                                   |
++=============================+==========================+======================================+===============================================================+
+|``43``                       |``recipient``             |``byte[43]``                          |The raw encoding of an Orchard Raw Payment Address, as         |
+|                             |                          |                                      |per protocol §5.6.4.2 ‘Orchard Raw Payment Addresses’.         |
++-----------------------------+--------------------------+--------------------------------------+---------------------------------------------------------------+
+|``8``                        |``value``                 |``uint64``                            |The amount of the Asset being issued in this note.             |
```
Comment (PaulLaux) [inline zip-0230.rst:348]: `The amount being issued in this note.`

**Review by PaulLaux — APPROVED** [review]
Very nice, added small comments

File: zip-0230.rst (hunk 2 above)
Comment (vivek-arte) [reply zip-0230.rst]: Done

File: zip-0230.rst (hunk 1 above)
Comment (vivek-arte) [reply zip-0230.rst]: Added that in all the rows it was missing.

**Pull request merged**